e2e/validators.go

```diff
@@ -1086,3 +1085,25 @@ fi`)},
 
 	s.T.Logf("PubkeyAuthentication is properly disabled as expected")
 }
+
+func ValidateNvidiaDCGMExporterSystemDServiceRunning(ctx context.Context, s *Scenario) {
+	s.T.Helper()
+	command := []string{
+		"set -ex",
+		// Verify nvidia-dcgm service is running
+		"systemctl is-active nvidia-dcgm",
+		// Verify nvidia-dcgm-exporter service is running
+		"systemctl is-active nvidia-dcgm-exporter",
+	}
+	execScriptOnVMForScenarioValidateExitCode(ctx, s, strings.Join(command, "\n"), 0, "Nvidia DCGM Exporter service validation failed")
+}
+
+func ValidateNvidiaDCGMExporterIsScrapable(ctx context.Context, s *Scenario) {
+	s.T.Helper()
+	command := []string{
+		"set -ex",
+		// Check if nvidia-dcgm-exporter is scrapable on port 19400
+		"curl -f http://localhost:19400/metrics",
```

Comment:
like Ganesh mentioned, maybe we should just ensure that we get back a format that is open-telemetry compatible.

Reply:
Yep added another a separate validator test for this, which ensures that the most common metric is available.